operations/reservations.md

```diff
@@ -74,7 +74,9 @@ Returns all reservations specified by any identifier, customer or other filter.
 | :-- | :-- | :-- | :-- |
 | `BusinessSegments` | bool | optional | Whether the response should contain business segmentation. |
 | `Customers` | bool | optional | Whether the response should contain customers of the reservations. |
-| `Items` | bool | optional | Whether the response should contain reservation items. |
+| `CustomersAddresses` | bool | optional | Whether the response should contain customer adresses. |
+| `CustomerIdentityDocuments` | bool | optional | Whether the response should contain customer identity documents. |
```

Comment:
those two were hidden in purpose. Those are deprecated. It's 1:m relationship that is not ideal and we want to avoid.

Reply:
okay, I wanted to discuss that with you. Fine, I'll remove those. Thanks.